services/authentication/logout.rb

```diff
@@ -0,0 +1,16 @@
+class AuthenticationService
+  class Logout < ApplicationService
+    def initialize(env)
+      @env = env
+    end
+
+    def call
+      env['warden'].raw_session.inspect
```

Comment:
Why do you need this line?

Reply:
removed it in last commit. Don't know why it's still appearing here